.gitignore

```diff
@@ -13,3 +13,6 @@ public
 resources
 node_modules
 generated
+install
+samples
+out
```

Comment:
Can you add a comment: # Contains the built artifacts

Reply:
Done.